### ISSUE TITLE
using touch to move in 2d not only 1d

### ISSUE THREAD

**Issue by sajadabasi** [lightbasenl/react-native-panorama-view#3]
Hello,
I want to use touch to move in two dimensions not only once! how can I do this?
Is it possible to disable sensor movements?
Thank you.

**Comment by rodymolenaar** [comment]
I think this is a duplicate of #5
